projects/portal/src/app/pages/delegate/validators/validator/validator.usecase.service.ts

```diff
@@ -38,23 +46,13 @@ export class ValidatorUseCaseService {
           const share = Number(validator.tokens) / allTokens;
           const statusBonded = validator.status === 'BOND_STATUS_BONDED';
           const inList = statusBonded && rank <= 50;
-          return { val, share, inList, rank };
+          return { val, share, inList, rank } as validatorType;
```

Comment:
Why is the type assertion here necessary?

Reply:
Sometime, this code can't get a type of "validatorsWitSd", so the prettier output an error. So I added this assertion. (sometime, not usually)